### PR TITLE
use stake address instead of stake public key in cip36 metadata

### PR DESCRIPTION
**Pull request by yushih**

### CODE REVIEW

File: packages/yoroi-extension/app/api/ada/lib/cardanoCrypto/catalyst.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export function generateRegistrationMetadata(
   const registrationData = RustModule.WalletV4.encode_json_str_to_metadatum(
     JSON.stringify({
       '1': [[prefix0x(votingPublicKey), 1]],
-      '2': prefix0x(stakingPublicKey),
+      '2': prefix0x(stakingAddress),
```
Comment (stackchain) [inline packages/yoroi-extension/app/api/ada/lib/cardanoCrypto/catalyst.js:46]: @michaeljscript can you check on mobile?

Comment (michalszorad) [reply packages/yoroi-extension/app/api/ada/lib/cardanoCrypto/catalyst.js:46]: @stackchain I'm having a look

Comment (michalszorad) [reply packages/yoroi-extension/app/api/ada/lib/cardanoCrypto/catalyst.js:46]: We're using `stakingPublicKeyHex`

Comment (michalszorad) [reply packages/yoroi-extension/app/api/ada/lib/cardanoCrypto/catalyst.js:46]: The implementation is in yoroi-lib of `generateRegistrationMetadata`. Can we reuse it in extension?

**Comment by yushih** [comment]
no need for this. Should be stake pub key.

File: packages/yoroi-extension/app/api/ada/lib/cardanoCrypto/catalyst.js (hunk 1 above)
Comment (yushih) [reply packages/yoroi-extension/app/api/ada/lib/cardanoCrypto/catalyst.js:46]: False change. Sorry. Things are correct as is.

**Pull request closed without merging**